README.md

```diff
@@ -50,7 +60,9 @@ Suppose you had the following form in your HTML file. Use jQuery to write a sing
 ```
 
 ### Question 7
-This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code? 
+This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code?
+
+  The code should define data, currently it is undefined so the console log is displaying the undefined data. Define data and the console log should display the data for the function.
```

Comment:
Not quite. The best solution is to move `UserApp.add_all_users(data);` inside the `.done()` callback.